index.html

```diff
@@ -502,29 +502,39 @@ <h2 class="section-heading">Contact</h2>
 		</div>
 	</footer>
 
+	<!-- Library JavaScript -->
+	<!--<script src="dist/js/lib.min.js"></script>-->
+
 	<!-- jQuery -->
+	<!-- <script src="js/jquery.js"></script>-->
+	<!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script> -->
 	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	<!-- <script type="text/javascript" src="https://www.murphyauto.com/_js/jquery-migrate-3.0.0.min.js"></script>
+    <script type="text/javascript" src="https://www.murphyauto.com/_js/jquery-migrate-1.4.1.min.js"></script> -->
 
 	<!-- Bootstrap Core JavaScript -->
-	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha256-KXn5puMvxCw+dAYznun+drMdG1IFl3agK0p/pqT9KAo= sha512-2e8qq0ETcfWRI4HJBzQiA3UoyFk6tbNyG+qSaIBZLyW9Xf3sWZHN/lxe9fTh1U45DpPf07yj94KsUHHWe4Yk1A==" crossorigin="anonymous"></script>
-
+	<!-- <script src="js/bootstrap.min.js"></script> -->
+	<!-- <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha256-KXn5puMvxCw+dAYznun+drMdG1IFl3agK0p/pqT9KAo= sha512-2e8qq0ETcfWRI4HJBzQiA3UoyFk6tbNyG+qSaIBZLyW9Xf3sWZHN/lxe9fTh1U45DpPf07yj94KsUHHWe4Yk1A==" crossorigin="anonymous"></script> -->
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
 	<!-- Plugin JavaScript -->
 	<!-- REVIEW TO FIGURE OUT WHY I CAN'T USE CDN VERSION -->
 	<script src="js/jquery.easing.min.js"></script>
+	<!-- <script src="js/jquery-ui.min.js"></script> -->
```

Comment:
Can we remove? I can do with the inline editor, but not sure why you would have added this...